### ISSUE TITLE
Maybe also add a recipe in conda-forge

### ISSUE THREAD

**Issue by valgur** [estnltk/conda-recipes#1]
Hi! Thanks for creating conda recipes for EstNLTK and its dependencies. It makes the installation significantly easier, especially on Windows.

Have you considered also adding a recipe in [conda-forge](https://conda-forge.github.io/)? In case you have not seen or used it before, I recommend taking a look. It's pretty much a near-official channel for contributed conda packages. I personally always configure conda to look for packages on conda-forge in addition to the default channel and there are probably many others. Each recipe is maintained there as a separate GitHub repo (e.g. https://github.com/conda-forge/python-crfsuite-feedstock) and new recipes can be contributed by creating pull requests to https://github.com/conda-forge/staged-recipes.

The pros of using conda-forge would be
* A tiny bit more easier installation through conda. Also makes it easier to use `estnltk` as a dependency in any other conda recipes.
* Building and releasing are taken care of automatically on continuous integration servers. Builds are created for Windows, Linux and Macs.
* Any configured tests are also run on all of the above platforms before creating a release.

The downsides
* No direct control over dependencies. On the positive side, there is no need to do any updating and maintenance on most of the dependencies, since they get updated by their respective maintainers.
* Possibly more work needed to maintain both the recipes here and on conda-forge.

I actually even went ahead and already created a potential recipe for EstNLTK about a month ago on `staged-recipes`: https://github.com/conda-forge/staged-recipes/pull/1750. That was before I noticed that conda builds are already available here. I created that recipe to generate a Python 3.5 build for `estnltk` which was also missing at that time.

That recipe is currently failing in the CI because of the broken Python 3.5 `python-crfsuite`, which I was trying to address here https://github.com/conda-forge/python-crfsuite-feedstock/pull/1.

If you are interested in contributing to the above pull requests, I'd gladly give access to my forks of these repos. I can also close the pull request for `estnltk` recipe if you happen to prefer that it would not simultaneously exist in the conda-forge ecosystem.

**Comment by urdvr** [comment]
Hi, 
I'm mostly responsible for maintaining conda recipes for the time being. 
I do hope to get  estnltk on conda-forge at some point, but currently the point about having direct control over dependencies is a sore one for us. 

Currently we have a version of python-crfsuite for windows builds and starting from the next release we'll be maintaining pyahocorasick (that isn't in conda-forge yet). In the medium-term I'd like to get [visl cg3](https://visl.sdu.dk/cg3.html) on our channel. I'm not sure I would have the time and energy to properly maintain all these and get any targeted fixes through the conda-forge pipeline (for example, the pull request you made to python-crfsuite-feedstock seems to have gone unnoticed by the maintainer for almost two weeks by now).

Most of our dependencies are pulled from conda-forge though. The recommended way to install estnltk right now would be `conda install -c estnltk -c conda-forge estnltk`.

In conclusion:
* I don't have any short- or medium-term plans of getting estnltk on conda-forge. I do hope to have it there in the long term.
* We don't currently have enough manpower to maintain everything we'd need to maintain on conda-forge. Most of the patches or fixes we apply on our own channel are fire-and-forget (and hope they eventually reach conda-forge).
* I don't mind if you attempt to maintain a conda-forge recipe for estnltk, but we'll keep advertising  anaconda.org/estnltk as the official channel right now.
